gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStoragePathTest.java

```diff
@@ -133,24 +132,24 @@ public void testGetName() {
   }
 
   @Test
-  public void testGetName_negative_throwsIae() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
+  public void testGetName_negative_throwsIae() throws IOException {
+    try (CloudStorageFileSystem fs = helper.forBucket("doodle")) {
       thrown.expect(IllegalArgumentException.class);
       fs.getPath("angel").getName(-1);
     }
   }
 
   @Test
-  public void testGetName_overflow_throwsIae() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/mziccard/gcloud-java/file/2333823945/issues/source?bid=3123107&fileBranchId=3190440#l143)